src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -49,6 +49,13 @@ const buttonStyles: IComponentPartStylesInput = {
       verticalAlign: 'middle',
       cursor: 'pointer',
 
+      ':focus': {
+        ...(isLastFocusFromKeyboard &&
+          {
+            // focus styles should be added like this, since they should be applied only on keyboard.
+          }),
+      },
```

Comment:
Should we merge this into this PR https://github.com/stardust-ui/react/pull/165 or we want to have this on the master so that it can be picked up for other styles as well?

Reply:
@mnajdova it was decided on the standup, that first will go @silviuavram PR, then Andrew will make his changes with styles. @silviuavram correct me if I am wrong :)

Reply:
Disregard this, I was just updated that that PR will be replaced with a new one.

Reply:
@sophieH29 I will not correct you because you are not wrong.